src/test/kotlin/no/nav/familie/ef/sak/mapper/AdresseHjelperTest.kt

```diff
@@ -47,6 +48,25 @@ internal class AdresseHjelperTest {
             assertThat(AdresseHjelper.harRegistrertSammeBostedsadresseSomForelder(barn, forelderAdresser)).isTrue
         }
 
+        @Test
+        internal fun `forelder og barn bor på samme adresse også med brukenhetsnummer null og tom streng `() {
```

Comment:
Kanskje greit med en test at to med "lik" adresse, men forskjellig bruksenhetsnummer ikke får "sammeAdresse" ? 
Altså;
Fyrstikkaleen, 1, H0101 != Fyrstikkaleen, 1, H0102